functions.lua

```diff
@@ -487,6 +487,12 @@ end
 
 -- Teleport Request System
 function tp.tpr_send(sender, receiver)
+	-- Check if the sender is the receiver
```

Comment:
L491 is self-explaning. Better document *why* a check like this is needed.
Example wording:
```suggestion
	-- Disallow entering inaccessible areas. (Teleportation includes a position offset of 1)
```

Same comment for L597 below.

Reply:
> Same comment for L597 below.